src/protocol.rs

```diff
@@ -23,11 +23,11 @@ pub enum Response {
     /// Response for `Request::Compile`.
     Compile(CompileResponse),
     /// Response for `Request::GetStats`, containing server statistics.
-    Stats(ServerInfo),
+    Stats(Box<ServerInfo>),
```

Comment:
Is this (and the `DistClientState` changes related to `Box`) due to `large_enum_variant`?

Reply:
yes